airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -326,56 +308,47 @@ def get_dag_asset_queued_events(
 
 
 @assets_router.get(
-    "/dags/{dag_id}/assets/queuedEvents/{uri:path}",
-    responses=create_openapi_http_exception_doc(
-        [
-            status.HTTP_404_NOT_FOUND,
-        ]
-    ),
+    "/dags/{dag_id}/assets/{asset_id:int}/queuedEvents",
```

Comment:
Nit: I don't think you need the explicit type in the different path definitions. FastAPI will read directly from type annotation.
```suggestion
    "/dags/{dag_id}/assets/{asset_id}/queuedEvents",
```

Reply:
Fixed (all of them)